documentation/DynamoDB and Codable.md

```diff
@@ -40,12 +48,16 @@ let input = DynamoDB.PutItemInput(
 )
 let output = try dynamoDB.putItem(input).wait()
 ```
+
 can be reduced to 
+
 ```swift
 let input = DynamoDB.PutItemCodableInput(item: person, tableName: "my-table")
 let output = try dynamoDB.putItem(input).wait()
 ```
-Similarly when using `DynamoDB.GetItem` the code below which parses an `AttributeValue` dictionary returned from `GetItem`
+
+Similarly when using `DynamoDB.GetItemInput(key:tableName:)`, the code below which parses an `AttributeValue` dictionary returned
```

Comment:
DynamoDB.getItem is the function. DynamoDB.GetItemInput is the input type to that function. I would suggest we use the function name.